gateway-messages/src/sp_to_mgs.rs

```diff
@@ -189,7 +208,13 @@ pub struct RotBootState {
     pub slot_b: Option<RotImageDetails>,
 }
 
-// TODO(AJS): Fill in with runtime state - i.e. updates that have completed before an RoT reset
+/// The currently active boot slot, as well as persistent and transient boot
```

Comment:
This docstring doesn't seem to describe an empty `struct RotBootInfo {}`

Reply:
Leftover. Removed in https://github.com/oxidecomputer/management-gateway-service/pull/92/commits/1879755cbe398073368338d11717a8ac8dc7fd93